manifests/dropin_file.pp

```diff
@@ -65,8 +69,10 @@
     }
   }
 
+  $full_filename = "${path}/${unit}.d/${filename}"
+
   if $ensure != 'absent' {
-    ensure_resource('file', "${path}/${unit}.d", {
+    ensure_resource('file', dirname($full_filename), {
```

Comment:
Is that because `$filename` might contain a slash?

Reply:
It was mostly so that I could really use the variable `$full_filename` everywhere.

Reply:
Ack.